contrib/osx/notes_for_arm.md

```diff
@@ -0,0 +1,56 @@
+# Notes on building Electrum on Apple M1 OSX (ARM)
```

Comment:
```suggestion
# Notes on running Electrum from source on ARM-based Macs (Apple M1 OSX)
```

Looks like the title is not technically correct - the text is re how to run from source on apple m1.
Building suggests it is about creating binaries (such as the .dmg files we distribute).